pages/thanks-rider.html

```diff
@@ -8,17 +8,28 @@
     <div class="wrapper single-column offset-top">
         <div class="bannerbox">
             <div class="bannerbox__content">
-                <h1>Congratulations</h1>
-                <p>on taking this step to claim your vote! We’ve sent you a message confirming that we’ve received your request.</p>
-                <p>Your ride request reference number is: <b class="uuid">ID not loaded, please try again</b>.</p>
-                <p>Please <strong>keep this reference</strong> in case you need to manage your ride request.</p>
-                <p>To view or manage your ride request, visit our <b><a class="self-service-url" href="../self-service/?type=rider">self-service portal</a></b>.</p>
+                <h1 id="thanks-header">Congratulations</h1>
+                <span id="sorryHeading" hidden style="display:none">We are sorry</span>
+                
+                <p id="introPara">on taking this step to claim your vote! We’ve sent you a message confirming that we’ve received your request.</p>
+                <span id="sorryPara" hidden style="display:none">It has not been possible to process your request. Please see the information below for more details. We want to help you, so please contact our phone support if you cannot resolve this issue yourself.</span>
```

Comment:
I see the pattern you suggest. I have deliberately chosen to use spans on purpose for their lack of meaning. Their purpose is to store alternate text, not to ever become visible (like data-* attributes in modern browsers). My goal here is to patch a fairly serious flaw, with some reasonable code that may be a work in progress/likely to have future amendments and move on to other tasks. 

Have you tested the page and seen what it does? I’m reluctant to refactor it before it’s been reviewed because it may all change again anyway. I took the extra step of putting the alternate text in the html rather than hard-coded in the js so that those on the team not so able to work with js could contribute to this (plus it’s more elegant). 

My preference is to merge it as it is, and I will happily wave on through any refactoring you wish to make at any point.